tests/DCPS/Compiler/annotations/is_topic_type_test/run_test.pl

```diff
@@ -0,0 +1,84 @@
+eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
+     & eval 'exec perl -S $0 $argv:q'
+     if 0;
+
+use strict;
+use JSON::PP;
```

Comment:
This doesn't seem like it will work with older Perl versions.  It would at least need a check to not cause an error if this module isn't found.

Reply:
Okay, do you want it to do a false pass if the module isn't available?